### DISCUSSION TITLE
Seem to be a bug : When file is not exist will show two identical rows prompt.

### DISCUSSION THREAD

**Discussion by sjh024** [ImageMagick/ImageMagick#7807] [Help]
### ImageMagick version

7.1.1-41

### Operating system, version and so on

Windows 10 64-bit

### Description

magick  %02d.jpg[1-5]   abcd.pdf 

When file is not exist will show two identical rows prompt.

![2024-11-25_211113](https://github.com/user-attachments/assets/d8e458fa-cd6a-4116-b2a6-22fb18ed3a80)

**Comment by urban-warrior** [comment]
Thank you for reporting the issue. We have successfully reproduced it and are actively working on a patch to resolve it. You can expect this patch to be merged into the [main GIT branch](https://github.com/ImageMagick/ImageMagick), later today. As part of our commitment to quality, this fix will also be included in the upcoming [beta releases of ImageMagick](https://imagemagick.org/archive/beta/) by tomorrow. Your patience and feedback are greatly appreciated.
